_posts/2017/2017-02-20-creating-a-sales-analytics-app-with-ruby-on-rails.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: "Creating a Sales Analytics App with Ruby on Rails"
+date: "2017-02-20 07:00:00"
+image: blog-header/hamburg-port-rails.jpg
+categories: tech-stories
+authors: ["Unai M."]
+---
+
+One of the languages that we use the most on the ePages Tools Team is [Ruby](https://www.ruby-lang.org).
+We love the syntax, the simplicity and the big and active community around it.
+Some time ago, [we wrote about our SDK](https://developer.epages.com/blog/2016/02/02/on-the-way-to-a-ruby-sdk.html) for the ePages shops and lots of things have changed since then.
+We've kept developing it to support more operations, such as data pagination and also to keep up with the releases of our colleagues from the [API development team](https://developer.epages.com/apps).
+
+## Status of the SDK
+
+We are preparing a release Soon&#8482; containing lots of model changes, in order to make the use of [the gem](https://github.com/ePages-de/epages-rest-ruby) even easier.
```

Comment:
It's a joke http://www.urbandictionary.com/define.php?term=soon%E2%84%A2 but if you think it's not nice to include this things, we can just change it to a simple soon